angular_token_auth.js

```diff
@@ -297,13 +298,20 @@
             }
         };
 
-        //use cookies if available, otherwise try localstorage
-        if (storageMethods['cookie'].test() === true) {
+        if (MODULE_SETTINGS.STORAGE_METHOD === 'cookie') {
             return storageMethods['cookie'];
-        } else if (storageMethods['localStorage'].test()) {
+        } else if (MODULE_SETTINGS.STORAGE_METHOD === 'localstorage') {
```

Comment:
At that point, the value of `'auto'` simply becomes a moniker for an unavailable storage method.